runtime/src/accounts_db.rs

```diff
@@ -354,13 +361,11 @@ pub struct AccountsDB {
     /// Keeps tracks of index into AppendVec on a per slot basis
     pub accounts_index: RwLock<AccountsIndex<AccountInfo>>,
 
-    /// Account storage
```

Comment:
Remove this as the same reason as [this](https://github.com/solana-labs/solana/pull/7559/files#r359688063).